pkg/apb/client.go

```diff
@@ -185,16 +171,13 @@ func (c *Client) PullImage(imageName string) error {
 	return nil
 }
 
-func (c *Client) refreshLoginToken(clusterConfig ClusterConfig) error {
-	c.log.Debug("Refreshing login token...")
-	c.log.Debug("target: [ %s ]", clusterConfig.Target)
-	c.log.Debug("user: [ %s ]", clusterConfig.User)
-	c.log.Debug("password:[ %s ]", clusterConfig.Password)
+func (c *Client) Login() error {
+	c.log.Debug("Logging in to openshift...")
 
 	output, err := runCommand(
-		"oc", "login", "--insecure-skip-tls-verify", clusterConfig.Target,
-		"-u", clusterConfig.User,
-		"-p", clusterConfig.Password,
+    "oc", "login", "https://kubernetes.default",
+    "--certificate-authority", "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt",
+    "--token", "$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)",
```

Comment:
I actually hadn't thought about this, I thought you were asking about openshift vs k8s. @rthallisey brings up a good point. Throws a wrench into broker development workflows. I wonder if there's a way to fake this out while running locally?